docs/CRSF.md

```diff
@@ -40,36 +45,35 @@ Configuration of MAVLink stream rates
 - SRx_POSITION = 2
 - SRx_RAW_SENS = 0 (for most of you this one is unimportant, keep it at 0 unless you really need it)
 
-Configuration for CRSF receiver
+Configuration for CRSF Receiver:
 
-Setting up ArduPilot for a CRSF receiver can be a bit tricky, as it can depend on the used board, and might need BRD_ALT_CONFIG to be set to a specific value. It is best to consult the ArduPilot wiki, or ask in the ArduPilot discussion channel.
+Setting up ArduPilot for a CRSF receiver can be a bit tricky, as it depends on the flight controller board, and might need BRD_ALT_CONFIG to be set to a specific value. It is best to consult the ArduPilot wiki, or ask in the ArduPilot discussion channel.
 
 For my Matek H743 board the configuration is:
 
 - BRD_ALT_CONFIG = 1
 - RC_PROTOCOLS = 536 or 512
-- SERIAL7_BAUD = irrelevant
+- RSSI_TYPE = 3
+- SERIAL7_BAUD = Irrelevant (Baud Rate is autodetected by Ardupilot)
 - SERIAL7_OPTIONS = 0
 - SERIAL7_PROTOCOL = 23
 
+There are more options available in the 'RC_OPTIONS' parameter: https://ardupilot.org/plane/docs/parameters.html#rc-options-rc-options
 
 ## mLRS Rx Module Setup
 
-These configurations are not strictly neccesary, but recommended.
+These configurations are not strictly neccesary, but recommended for Ardupilot:
 
 - Rx Out Mode = crsf
+- RX Ser Baudrate = 57600
 - Rx Ser Link Mode = mavlink
-- Rx Snd RadioStat = w txbuf
-
-
-## OpenTx and Yaapu Telemetry App
-
-The radio needs to be set up for CRSF. This however proceeds exactly as described in common tutorials.
-
-In EdgeTx/OpenTx go as usual to MDL->TELEMETRY and start "Discover new sensors". You should then see plenty of sensors appearing.
+- Rx Snd RadioStat = ardu_1
```

Comment:
the problem here is if the docu shall be for the latest release, or for the latest dev version

do we want it for release or dev?

Reply:
Good point, I added nested bullets for 'Rx Snd RadioStat' which has the value for versions greater than and less than 0.3.13.

Reply:
well, the problem is quite of general nature ... I mean, it may work out now, but 10 official and 20 dev versions later it may get out of control ...
at some point in the future a decission needs to be made :)

Reply:
I propose that we keep as is for the moment and then with next release (0.4) can stick to the release version going forward.